### DISCUSSION TITLE
Speaker and Emotion Model Released?

### DISCUSSION THREAD

**Discussion by flckv** [speechbrain/speechbrain#1971] [Q&A]
-

**Comment by Adel-Moumen** [comment]
Hi, yes there are available through the links provided in their README.md e.g. https://github.com/speechbrain/speechbrain/tree/develop/recipes/IEMOCAP 

You can also download them directly through HuggingFace e.g. https://huggingface.co/speechbrain/emotion-recognition-wav2vec2-IEMOCAP
